packages/react-image/src/components/Image/useImage.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { mergeProps, getSlots, resolveShorthandProps } from '@fluentui/react-compose/lib/next/index';
+import { ImageProps, ImageState } from './Image.types';
+import { useImageState } from './useImageState';
+import { useMergedRefs } from '@uifabric/react-hooks';
+
+/**
+ * Consts listing which props are shorthand props.
+ */
+export const iconShorthandProps = [];
+
+/**
+ * Define the render function. Given the state of a button, renders it.
+ */
+export const renderImage = (state: ImageState) => {
+  const { slots, slotProps } = getSlots(state, iconShorthandProps);
```

Comment:
Do we really need this for a component that does not have any slots? 🤔

Reply:
I believe we still need it as we need the resolved `slots.root`, as well as the `slotProps.root`

Reply:
Yep! It boils down to:

* slot.root resolution
* slotProps.root resolution (if primitive, getNativeProps. Otherwise pass through props and assume the component will do the filter.)